docs/examples/viz_slicer.py

```diff
@@ -66,58 +69,64 @@ def handle_key_event(event):
     elif event.key == "ArrowDown":
         position -= 1
 
-    position = np.maximum(np.zeros((3,)), position)
-    position = np.minimum(np.asarray(data.shape), position)
+    bounds = slicer_actor.get_bounding_box()
+
+    position = np.maximum(bounds[0], position)
+    position = np.minimum(bounds[1], position)
     show_slices(slicer_actor, position)
     show_m.render()
 
 
-###############################################################################
-# Add event handlers to the slicer actor for picking and key events.
-
-slicer_actor.add_event_handler(handle_pick_event, "pointer_down")
+################################################################################
+# Add event handlers to the slice actor for key events.
 
-show_m = window.ShowManager(scene=scene, title="FURY 2.0: MNI Template Slicer Example")
+show_m = window.ShowManager(scene=scene, title="FURY 2.0: Cube Slicer Example")
 show_m.renderer.add_event_handler(handle_key_event, "key_down")
 
 if __name__ == "__main__":
     show_m.start()
```

Comment:
it is strange to have this twice ?

Reply:
We have two different windows demonstrating first the regular slicer and the other one is volume slicer.

I missed something to revert while testing with another dataset.